bambam.py

```diff
@@ -69,15 +69,15 @@ def play(self): pass
     return sound
 
 
-# Loads a list of sounds
-def load_sounds(lst):
+# Runs load_function on elements of lst unless they are blacklisted.
```

Comment:
The doctring convention is not used in the rest of the code.
I porpose to migrate to doctring convention for all methods in a dedicated commit, in another pullrequest.
What is your opinion ?